.idea/modules.xml

```diff
@@ -3,6 +3,7 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/DSBridge.iml" filepath="$PROJECT_DIR$/DSBridge.iml" />
+      <module fileurl="file://$PROJECT_DIR$/DSBridge-Android.iml" filepath="$PROJECT_DIR$/DSBridge-Android.iml" />
```

Comment:
we should add this folder to gitignore,   .idea

Reply:
ok